mailing.go

```diff
@@ -38,7 +32,13 @@ type Request struct {
 	to      []string
 	subject string
 	body    string
-	super   []byte
+}
+
+type SendEmailsRequest struct {
```

Comment:
You should also put the sender in field rather than writing it in `SendEmails` (actuel function name).